project_management/SSI_blog_06_03_19.md

```diff
@@ -0,0 +1,73 @@
+- Version control ([ready for community feedback](https://github.com/alan-turing-institute/the-turing-way/blob/master/chapters/version_control.md))
+- Open research ([ready for community feedback](https://github.com/alan-turing-institute/the-turing-way/blob/master/chapters/open_research.md))
+- Sustainable research data management (in progress)
+- Testing (in progress)
+- Reproducible computational environments (in progress)
+- Continuous integration (in progress)
+- Continuous analysis (in progress)
+- Reproducibility with [interactive notebooks](https://jupyter.org/) (community help wanted)
+- Good coding practice (community help wanted)
+- Reproducibility with Deep Learning (community help wanted)
+
+Fundamentally this is an open project so everyone is encouraged to get involved! Whether that’s by making improvements to already complete chapters, writing new ones, reviewing new work, raising issues, or any other contribution it’s welcome and appreciated. We want this to be a living, evolving resource that serves and is supported by the wider community.
+
+## Case Studies of Reproducible Research in Practice
+
+We’re working with several [researchers](https://github.com/alan-turing-institute/ReproducibleResearchResources#announcement-turing-reproducible-research-champions) from the Alan Turing Institute (the Turing reproducible research champions) to reproduce their work, share their experiences and provide case studies of reproducible research in practice. We’re interested in why reproducibility is important to them, how they went about trying to work in a reproducible way, and crucially what advice they would give to their previous selves.   
+
+Once complete, these case studies will be will be interwoven throughout the textbook to demonstrate how theory translates to practice. They will allow readers to follow along with the work of the Turing reproducible research champions as we collaborated with them to overcome challenges including reproducible analysis when using HPC resources, managing and packaging fast-growing projects and sharing workflows built for sensitive data.
+
+If you have tips and tricks, or “gotchas” that caught you out as you tried to make your work reproducible, please share them through [this google form](https://docs.google.com/forms/d/e/1FAIpQLSfcyYH_E03Y5zuBdwrikQP4QldqzKmD-aPngCKthdV9e9alaA/viewform).
+
+## Workshops for Researchers and Research Software Engineers
+
+We’re running some workshops as part of the project. Two (one in Manchester on March 1st and one in [London on March 12th](https://www.eventbrite.co.uk/e/boost-your-research-reproducibility-with-binder-london-registration-55337162944)) was/will be on helping people to boost their research reproducibility with [Binder](https://mybinder.readthedocs.io/en/latest/). Binder is a tool for helping people share code without worrying about the computational environment they're running or installing a long list of requirements.
```

Comment:
Has this blog done out yet? Does this section need to be re-written in the past tense and to include some feedback from the Manchester event?

Reply:
Yes it's out. I've included the "was/will"s for Manchester/London

Reply:
Gotcha! Sorry - I missed the link on the top line.

Reply:
No problem!